### ISSUE TITLE
Contact Kubernetes to verify the training data

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#43]
## User story
1. As a develop team
2. I want / need to utilise the power of our industry partner
3. So that we invite them to perform double-check once our training data is ready

## Acceptance criteria
* Set up a meeting with Kubernetes 
* Invite their team to verify the training data when we are ready
* Organise the results into a presentable report (maybe from Kubernetes)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
